.gitignore

```diff
@@ -21,6 +21,10 @@ Binaries/*.pdb
 Binaries/*.config
 Binaries/*.vshost.exe.manifest
 
+# Coco/R
+Scanner.cs.old
```

Comment:
What are these `.old` files?

Reply:
Backup copies that Coco creates when the output files (`Parser.cs` and `Scanner.cs`) already exist.